### PR TITLE
This is a fix for #60

### PR DESCRIPTION
**Pull request by Yathannsh**
I previously pushed changes directly to the upstream, sorry about that xD

So the demo is the current website itself .
 I opened an issue and not sure about how to acknowledge this as a solution there, would be great if someone could help me out with that thanks!

### CODE REVIEW

File: _site/stylesheets/stylesheet.css [hunk 1]
```diff
@@ -56,9 +56,8 @@ html, body {
   color: #484848;
 }
 @media (max-width:768px) {
-
   .codein-bkg{
-    display: none;
+     display: none;
```
Comment (harshkothari410) [inline _site/stylesheets/stylesheet.css:60]: Remove extra space

File: stylesheets/stylesheet.css [hunk 2]
```diff
@@ -56,9 +56,8 @@ html, body {
   color: #484848;
 }
 @media (max-width:768px) {
-
   .codein-bkg{
-    display: none;
+     display: none;
```
Comment (harshkothari410) [inline stylesheets/stylesheet.css]: remove extra space

**Comment by Yathannsh** [comment]
@harshkothari410 thanks for correcting, I have removed the extra space!

**Comment by harshkothari410** [comment]
Let me review it.

**Comment by Yathannsh** [comment]
Thanks for merging @harshkothari410 .
I think I can help out with fixing the reaming issues as well. 
Mario told me to get tasks formulated for all the fixes, thus he made a collective task which you just approved. Is it possible to reclaim this task and fix the remaining issues or should I just send pull requests directly after I am done fixing them?

**Pull request merged**